### PR TITLE
✨ FEAT: AggregateSparqlView class

### PR DESCRIPTION
**Pull request by kenjinp**
For all your aggregated sparql query needs

```typescript
const aggregateSparqlView = new AggregateSparqlView(
        'myOrg',
        'myProject',
        mockAggregatedSparqlViewGetResponse,
      );
```

### CODE REVIEW

**Comment by kenjinp** [comment]
Brings it up to par with the other views, but will add CRUD functionality afterwards following a refactor using `makeUtils`

File: src/View/AggregateSparqlView/__tests__/AggregateSparqlView.spec.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  _project: 'http://example.com/v1//v1/projects/my-org/my-project',
+  _rev: 1,
+  _deprecated: false,
+  _createdAt: '2019-04-10T08:58:39.530Z',
+  _createdBy: 'http://example.com/v1//v1/realms/github/users/kenjinp',
+  _updatedAt: '2019-04-10T08:58:39.530Z',
+  _updatedBy: 'http://example.com/v1//v1/realms/github/users/kenjinp',
+};
+
+const testMetaData = (
```
Comment (julienmachon) [inline src/View/AggregateSparqlView/__tests__/AggregateSparqlView.spec.ts]: use snapshots!

File: src/View/AggregateSparqlView/__tests__/AggregateSparqlView.spec.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+describe('AggregateSparqlView', () => {
+  describe('aggregateSparqlView', () => {
+    it('should have all the properties created by the response', () => {
+      const aggregateSparqlView = new AggregateSparqlView(
+        'myOrg',
+        'myProject',
+        mockAggregatedSparqlViewGetResponse,
+      );
+      console.log(aggregateSparqlView);
```
Comment (julienmachon) [inline src/View/AggregateSparqlView/__tests__/AggregateSparqlView.spec.ts]: :gun:

Comment (kenjinp) [reply src/View/AggregateSparqlView/__tests__/AggregateSparqlView.spec.ts]: 💣

File: src/View/AggregateSparqlView/__tests__/AggregateSparqlView.spec.ts (hunk 1 above)
Comment (kenjinp) [reply src/View/AggregateSparqlView/__tests__/AggregateSparqlView.spec.ts]: oh yeah!

File: src/View/AggregateSparqlView/__tests__/AggregateSparqlView.spec.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      viewId: 'http://example.com/base/custom',
+    },
+  ],
+  _self: 'http://example.com/v1//v1/views/my-org/my-project/aggregated',
+  _constrainedBy: 'https://bluebrain.github.io/nexus/schemas/view.json',
+  _project: 'http://example.com/v1//v1/projects/my-org/my-project',
+  _rev: 1,
+  _deprecated: false,
+  _createdAt: '2019-04-10T08:58:39.530Z',
+  _createdBy: 'http://example.com/v1//v1/realms/github/users/kenjinp',
```
Comment (mdupont) [inline src/View/AggregateSparqlView/__tests__/AggregateSparqlView.spec.ts]: 2 things:
- do you really want you github username in the public code? ;-)
- there is `/v1//v1` in the URL? wrong prefix it seems

Comment (kenjinp) [reply src/View/AggregateSparqlView/__tests__/AggregateSparqlView.spec.ts]: thanks. Also I think my username is here anyways?

**Pull request merged**